src/main/java/ch/njol/skript/expressions/ExprKickMessage.java

```diff
@@ -0,0 +1,74 @@
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.util.Kleenean;
+import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.event.Event;
+import org.bukkit.event.player.PlayerKickEvent;
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.*;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Kick Message")
+@Description("The kick message that is displayed on-screen when a player is kicked.")
+@Examples({
+	"on kick:",
+		"\tkick message is 'Invalid hotbar selection (Hacking?)'",
+		"\tcancel event"
+})
+@Since("INSERT VERSION")
+@Events("Kick")
+public class ExprKickMessage extends SimpleExpression<String> {
+
+	static {
+		Skript.registerExpression(ExprKickMessage.class, String.class, ExpressionType.SIMPLE, "(disconnect|kick) message");
```

Comment:
```suggestion
		Skript.registerExpression(ExprKickMessage.class, String.class, ExpressionType.SIMPLE, "kick (message|reason)");
```
There's a difference between disconnecting and getting kicked, so I don't think `disconnect` should be in the pattern
Also, since the methods that are being utilized is **reason** should also allow `reason` in the pattern

Reply:
`kick message` would conflict with the existing `kick message` expression under ExprMessage